pom.xml

```diff
@@ -64,7 +64,7 @@
     <hamcrest-library.version>1.3</hamcrest-library.version>
     <housekeeping.version>1.0.4</housekeeping.version>
     <guava.version>20.0</guava.version>
-    <hcommon-hive-metastore.version>1.0.0</hcommon-hive-metastore.version>
+    <hcommon-hive-metastore.version>1.0.1</hcommon-hive-metastore.version>
```

Comment:
Won't this PR require a version of this that hasn't been released yet? i.e. this should be a SNAPSHOT until we release it and then this can be updated to that released version. I can't see how else you'd manage to get this to compile.